.rubocop.yml

```diff
@@ -149,7 +149,7 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
+  MaxLineLength: 80
```

Comment:
This is only for one-liner `if` and `unless` statements. When you have something like this:
```ruby
if some_long_condition_that_takes_up_many_characters
  do_something_that_also_takes_up_space
end
```
and if the one-liner will fit inside 100 characters, Rubocop will complain and say that you should do this:
```ruby
do_something_that_also_takes_up_space if some_long_condition_that_takes_up_many_characters
```
For one-liners, I think Rubocop should only complain if the one-liner fits inside 80 characters. Thoughts?